src/backend/renderer/mod.rs

```diff
@@ -103,27 +105,23 @@ impl From<wayland_server::protocol::wl_output::Transform> for Transform {
 }
 
 /// Abstraction for Renderers, that can render into different targets
-pub trait Bind<Target>: Unbind {
-    /// Bind a given rendering target, which will contain the rendering results until `unbind` is called.
+pub trait Bind<Target>: Renderer {
+    /// Initialize a framebuffer with a given rendering target.
     ///
-    /// Binding to target, while another one is already bound, is rendering defined.
-    /// Some renderers might happily replace the current target, while other might drop the call
-    /// or throw an error.
-    fn bind(&mut self, target: Target) -> Result<(), <Self as Renderer>::Error>;
+    /// The `output_size` specifies the dimensions of the display **before** the `dst_transform` is
+    /// applied.
+    ///
+    /// This function *may* error, if:
+    /// - The given dimensions are unsupported (too large) for this renderer
+    /// - The given Transformation is not supported by the renderer (`Transform::Normal` is always supported).
```

Comment:
Is this a left-over from the `render` refactor? If we have some constraints between `bind` and `render` we should describe them, but `output_size` is not part of this function signature.

Reply:
fixed